go.mod

```diff
@@ -6,17 +6,15 @@ require (
 	github.com/IBM/sarama v1.45.2
 	github.com/gorilla/mux v1.8.1
 	github.com/jedib0t/go-pretty/v6 v6.6.7
+	github.com/joho/godotenv v1.5.1
```

Comment:
you can load the variables from the env file via Makefile, see [here](https://github.com/IBM/opentalaria/blob/main/Makefile#L1) for example. We don't really have a use case for writing the env variables to an env file, so you can remove this library.